apps/builddao/widget/components/Navbar.jsx

```diff
@@ -286,13 +297,17 @@ function Navbar(props) {
                 <li>
                   <Link
                     style={{ textDecoration: "none" }}
-                    href={href({
-                      widgetSrc: "${config_account}/widget/app",
-                      params: {
-                        page: "inspect",
-                        widgetPath: routes[page].path,
-                      },
-                    })}
+                    href={
+                      !fromGateway
```

Comment:
What's the "fromGateway" variable for?

Reply:
This is because when we are using the gateway, the routing needs to be consistent. We don't want to be routing between widgets and clear routing i've setup.

Reply:
Ohhh I see. I wonder if this could be moved to the `<Link />` custom element in the gateway itself instead

Reply:
I thought so too, still we would have the hassle of passing in alternate route

Reply:
what alternative route?
I mean I would send the same href link which we did earlier only, and if it's from gateway we can parse it directly in that Link component
something like this,
so we pass
```
href={href({
  widgetSrc: "${config_account}/widget/app",
  params: {
  page: "inspect",
  widgetPath: routes[page].path,
  },
})}
 ```
 I know the page and path
 so I can do `page/widgetPath` if it's not from gateway
 or am I missing something?